src/compositemark/common.ts

```diff
@@ -117,7 +117,7 @@ export function getCompositeMarkTooltip(
       return {
         field: fieldPrefix + continuousAxisChannelDef.field,
         type: continuousAxisChannelDef.type,
-        title: isSignalRef(titlePrefix) ? {signal: titlePrefix + mainTitle} : titlePrefix + mainTitle
+        title: isSignalRef(titlePrefix) ? {signal: titlePrefix + `"${mainTitle}"`} : titlePrefix + mainTitle
```

Comment:
Shouldn't this be

```js
{signal: titlePrefix.signal + ` + "${escape(mainTitle)}"`}
```

where `escape` should be something that replaces `"` with its escaped version.

Reply:
fixed